src/main/java/seedu/sellsavvy/model/order/Order.java

```diff
@@ -60,6 +60,13 @@ public boolean isSameOrder(Order otherOrder) {
                 && otherOrder.getItem().equals(getItem());
     }
 
+    /**
+     * Returns {@code Date} warning if order date has passed.
+     */
+    public String hasDateElapsed() {
```

Comment:
Same applies here for the earlier comment regarding returning boolean.

Reply:
Updated in commit.